R/repr_htmlwidget.r

```diff
@@ -22,3 +22,57 @@ repr_html.htmlwidget <- function(obj, ...) {
 	
 	readChar(htmlfile, file.info(htmlfile)$size)
 }
+
+
+
+# sort of like htmlwidgets::saveWidget(), but for HTML tags
+save_html <- function(obj, ...) {
+	
+	if (!requireNamespace('htmlwidgets')) {
+		stop('Please install the htmlwidgets package')
+	}
+	if (!getFromNamespace("pandoc_available", asNamespace("htmlwidgets"))()) {
```

Comment:
could you assign this and the other function you retrieve via `getFromNamespace` to variables please?

i think it’s cleaner to do `if (!pandoc_available())` than this expression